dipy/viz/ui.py

```diff
@@ -1079,8 +1079,15 @@ def __init__(self, width, height, text="Enter Text", position=(100, 10),
         """
         super(TextBox2D, self).__init__()
         self.text = text
-        self.actor = self.build_actor(self.text, position, color, font_size,
-                                      font_family, justification, bold, italic, shadow)
+        self.actor = TextBlock2D(self.text, font_size, font_family,
+                                 justification, bold, italic, shadow, color,
+                                 position)
+        if vtk.vtkVersion.GetVTKSourceVersion().split(' ')[-1] <= "6.2.0":
```

Comment:
Can you create a function `_build()` and place the background logic inside it? I know it's kind of redundant, but we have been doing things that way for every UI element. Also, in future, if we are to add some more modifications to the text block at the text box level, we might end up complicating the constructor, so a separate build function makes more sense.

Reply:
Do you have other changes in mind that are specific to `TextBox2D` or `FileSelect2D`? If not, I think we could clean this up better by moving background color, background opacity, and line spacing to the `TextBlock2D` constructor. Those are probably common enough settings to belong there.

Reply:
As of this moment, no.